apps/web/src/components/buttons/feedback-button.tsx

```diff
@@ -58,7 +64,9 @@ export default function FeedbackButton(): ReactNode {
           title: tGeneric('success'),
           message: t('successMessage'),
           color: 'blue',
+          position: 'top-center',
```

Comment:
```suggestion
          position: 'top-center',
          autoClose: 5000,
```
or maybe 4000, not sure